db/migrations/20191013190112_courses_table.js

```diff
@@ -1,6 +1,5 @@
 exports.up = (knex) => {
   const query = `
-
     create type user_role as enum('student', 'professor', 'admin');
 
     CREATE TABLE IF NOT EXISTS courses(
```

Comment:
check in the API definition document about the rest of the metadata

Reply:
for example, the guide name

Reply:
we should add a primary key for `id`

also, replace id for `course_id`

Reply:
also we should add a constraint for the pairs: `course_id`, `user_id` 

add a primary key for `guide_id`

Reply:
done!